process/setup_sp.py

```diff
@@ -373,6 +373,100 @@ def process_distant_nodes(
     return(distant_nodes)
 
 
+#Cumulative opportunities (binary)
+#1 if d <= access_dist
+#0 if d > access_dist
+def binary_access_score(df, new_name, old_name, access_dist):
+    """
+    Calculate accessibiity score using binary measure: 1 if access <= access_dist, 0 otherwise
+
+    Parameters
+    ----------
+    df: DataFrame
+        DataFrame with origin-destination distances
+    new_name: list
+        list of new field names for new access score
+    old_name: list
+         list of original field names for distances
+    access_dist: int
+        access distance threshold
+
+    Returns
+    -------
+    DataFrame
+    """
+    df[new_name] = (df[old_name] <= access_dist).astype("Int64").fillna(0)
+    return df[new_name]
+
+
+#Soft threshold access score
+#Higgs, C., Badland, H., Simons, K. et al. The Urban Liveability Index
+def soft_access_score(df, new_name, old_name, access_dist, k):

```

Comment:
Let's provide defaults for access_dist and k, matching our case study parameterisation:
`access_dist=500, k=5`